sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -938,7 +788,8 @@ export class ServiceBusMessageImpl implements ServiceBusReceivedMessage {
     if (msg.body) {
       this.body = this._context.dataTransformer.decode(msg.body);
     }
-    this._amqpAnnotatedMessage = toAmqpAnnotatedMessage(msg);
+    // TODO: I believe this is not needed, remove it
```

Comment:
Yes, a few lines above, we call `fromAmqpMessage()` which internally calls `fromRheaMessage()`

Reply:
Yeah, that's why I added the TODO.